lib/SimpleSAML/XML/Shib13/AuthnResponse.php

```diff
@@ -131,10 +125,11 @@ public function validate()
     }
 
 
-    /* Checks if the given node is validated by the signature on this response.
+    /**
+     * Checks if the given node is validated by the signature on this response.
      *
-     * Returns:
-     *  TRUE if the node is validated or FALSE if not.
+     * @param \DOMElement Node to be validated.
```

Comment:
The name of the parameter here (`$node`) is missing. I see phpdoc supports that, but they discourage the practice. It's not really a big issue, specially given this is a private method, but it would be nice to correct it 😊